lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -106,15 +106,24 @@ function _dosomething_northstar_build_http_client() {
  */
 function dosomething_northstar_update_user($form_state) {
   $user = $form_state['user'];
+  $id = $user->uid;
+
   $ns_user = dosomething_northstar_build_ns_user($user, $form_state);
 
   $northstar = new Northstar();
   $client = _dosomething_northstar_build_http_client();
-  $response = drupal_http_request($client['base_url'] . '/users', array(
+
+  $northstar_user_info = dosomething_northstar_get_northstar_user($id);
+  $northstar_user_info = json_decode($northstar_user_info->data, true);
+  $northstar_user_info = (object) $northstar_user_info['data'][0];
+  $ns_id = $northstar_user_info->_id;
+
+  $response = drupal_http_request($client['base_url'] . '/users/' . $ns_id, array(
     'headers' => $client['headers'],
-    'method' => 'POST',
+    'method' => 'PUT',
     'data' => json_encode($ns_user),
     ));
+
```

Comment:
I can see this function being reallly slow in prod since we are making two requests in a row.. can we do this with one request? I thought the `user` POST endpoint would search to see if the user exists before creating a new one?